packages/sanity/src/desk/panes/document/documentInspector/DocumentInspectorHeaderCard.tsx

```diff
@@ -0,0 +1,36 @@
+import {Card, CardProps} from '@sanity/ui'
+import React, {HTMLProps} from 'react'
+import styled from 'styled-components'
+
+const Root = styled(Card)({
+  position: 'relative',
+  zIndex: 1,
+  lineHeight: 0,
+
+  '&:after': {
+    content: '""',
+    display: 'block',
+    position: 'absolute',
+    left: 0,
+    bottom: -1,
+    right: 0,
+    borderBottom: '1px solid var(--card-border-color)',
+    opacity: 0.5,
+  },
+})
+
+/** @internal */
+export function DocumentInspectorHeaderCard(
```

Comment:
I think it would be nice if the close button was part of the `DocumentInspectorHeaderCard` as it would help maintain consistent design across the inspector headers. I suggest an update where`onClose` and`title` are added as props, and that `children` is an optional prop used in situations such as for e.g. the review changes inspector.

Example:

```ts
 <DocumentInspectorHeaderCard onClose={onClose} title="My inspector" />
```

```ts
<DocumentInspectorHeaderCard onClose={onClose} title="My inspector">
  <Content />
</DocumentInspectorHeaderCard>
```

Reply:
Addressed! And renamed the component to `DocumentInspectorHeader`.